components/title/basic/src/index.scss

```diff
@@ -0,0 +1,48 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$c-title-basic: $c-white;
+$c-subtitle-basic: $c-white;
+
+$ff-roboto-sans-serif: 'Roboto', sans-serif; // Will be moved to nc-theme-basic
```

Comment:
thinking about it, what about using the theme-basic titles definition?
https://github.com/SUI-Components/theme-basic/blob/master/src/_typography.scss